common/script/index.js

```diff
@@ -2279,13 +2279,14 @@ api.wrap = function(user, main) {
           user.stats.lvl++;
           tnl = api.tnl(user.stats.lvl);
           user.stats.hp = 50;
-          if (user.stats.lvl > api.maxLevel) {
+          var totalStatPoints = user.stats.str + user.stats.int + user.stats.con + user.stats.per;
+          if (user.stats.lvl > api.maxLevel && totalPoints >= 100 ) {
```

Comment:
Yep, i think you are right. The points seem to outweigh the level criteria. I'll make a change when I get home tonight unless anyone else thinks the check should stay.